src/components/pages/Chat/RoomNameLoader.tsx

```diff
@@ -20,13 +21,22 @@ export default function RoomNameLoader({ roomId }: Props) {
             }
 
             const stream = await streamrClient.getStream(roomId)
```

Comment:
Let's check if this effect is still relevant after this `await`, i.e. `let mounted = true`, etc.